pmd-apex/src/main/resources/category/apex/design.xml

```diff
@@ -281,7 +281,7 @@ public void addPerson(Date birthdate, BodyMeasurements measurements, int ssn) {
     <rule name="ExcessivePublicCount"
           language="apex"
           since="5.5.0"
-          message="This class has a bunch of public methods and attributes"
+          message="The class {0} has {1} public methods and attributes"
```

Comment:
the change is a net positive, but in retrospective, to make it even more useful / self-contained, maybe it should be something along the lines:

```xml
The class {0} has {1} public methods and attributes (limit: {2})
```

Otherwise, a dev trying to fix this would have to look into the ruleset to figure out what the property is set to… This is something we probably need to improve for most (all?) metric-based rules in all languages.